tools/parity_scale_cli/README.md

```diff
@@ -0,0 +1,124 @@
+# Parity Scale Decoder Tool
+
+This tool helps you decode **Iroha 2** data types from binaries using [Parity Scale Codec](https://github.com/paritytech/parity-scale-codec).
+
+## Build
+
+To build the tool, run:
+
+```bash
+cargo build --bin parity_scale_cli
+```
+
+If your terminal does not support colours, run:
+
+```bash
+cargo build --features no_color --bin parity_scale_cli
+```
```

Comment:
Instead of feature-gating it, I would suggest following the same pattern that `iroha` (crate in <kbd>/cli/</kbd>) itself follows:

- It detects colors support automatically via `supports_color` crate
- User can override it by `--terminal-colors=false`/`--terminal-colors=true` flag